src/storage.h

```diff
@@ -35,10 +36,28 @@ namespace cycamore {
 /// sell_quantity restricts selling to only integer multiples of this value
 /// max_inv_size is the maximum capacity of the inventory storage
 /// throughput is the maximum processing capacity per timestep
-/// active_buying is the number of time steps in a row where the agent 
-/// exhibits default behavior
-/// dormant_buying is the number of time steps in a row where the agent is 
-/// not requesting any new material
+/// active_buying_frequency_type is the type of distribution used to determine the length of the active buying period
+/// active_buying_val is the length of the active buying period if active_buying_frequency_type is Fixed
+/// active_buying_min is the minimum length of the active buying period if active_buying_frequency_type is Uniform (required) or 
```

Comment:
Code below says this is optional for Uniform - please confirm correct response

Reply:
hm good point. I think for simplicity's sake it's probably best to call them both required, making the user be explicit about what they want